app/controllers/verify/review_controller.rb

```diff
@@ -23,6 +23,13 @@ def confirm_current_password
     def new
       idv_session.params.symbolize_keys!
       analytics.track_event(Analytics::IDV_REVIEW_VISIT)
+
+      phone_of_record_msg = ActionController::Base.helpers.content_tag(
+        :strong, t('idv.messages.phone.phone_of_record')
+      )
+
+      flash[:now] = t('idv.messages.review.info_verified_html',
```

Comment:
Just noticed something after trying this out locally.  This should be `flash[:success]` not `flash[:now]` if you want it to appear green.  The name of the key is what is used to match it up with the corresponding css class.  And if you are wanting to do flash.now, it would be `flash.now[:success]`.